package.json

```diff
@@ -23,8 +23,7 @@
     "build": "npm run build:data && npm run build-css && next build",
     "build-css": "node-sass --include-path scss styles/index.scss static/index.css --output-style compressed && node-sass --include-path scss styles/success-banner.scss static/success-banner.css --output-style compressed",
     "serve": "next start",
-    "export": "npm run build && next export -o build && cp CNAME build/CNAME && cp .nojekyll build/.nojekyll && npm run copy-static",
-    "copy-static": "cp -r static_html/* build/",
+    "export": "npm run build && next export -o build && cp CNAME build/CNAME && cp .nojekyll build/.nojekyll",
```

Comment:
Should this simplification of the build-system be mentioned in the commit-message too?
(perhaps it'd be worth splitting this PR into 2 commits? \*shrug\*)